org.mixedrealitytoolkit.input/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "org.mixedrealitytoolkit.input",
-  "version": "3.1.0-development",
+  "version": "3.2.0-development",
```

Comment:
I think this package's Core dependency's version needs to be updated as well

Reply:
Ah, right, thanks for the catch!  Done.